packages/universal-cookie/src/utils.ts

```diff
@@ -3,14 +3,24 @@ import { Cookie, CookieGetOptions } from './types';
 
 export function hasDocumentCookie() {
   // JSDOM does not support changing cookies, disable it for tests
-  if (isJsDom()) {
+  if (isJsDom() || cookieDisabledBySettings()) {
     return false;
   }
 
   // Can we get/set cookies on document.cookie?
   return typeof document === 'object' && typeof document.cookie === 'string';
 }
 
+function cookieDisabledBySettings() {
+  if (typeof navigator !== 'object') {
+      return true;
```

Comment:
If `navigator` is not defined, it does not mean the cookies are disabled. It should default to false, not true otherwise it would change the behavior on server-side.